sktime/forecasting/online_ensemble.py

```diff
@@ -0,0 +1,150 @@
+import numpy as np
+import pandas as pd
+from sktime.forecasting.base._base import DEFAULT_ALPHA
+from sktime.forecasting.compose._ensemble import EnsembleForecaster
+from sktime.forecasting.model_selection import SlidingWindowSplitter
+from sktime.utils.validation.forecasting import check_cv
+from sktime.utils.validation.forecasting import check_y
+
+
+class OnlineEnsembleForecaster(EnsembleForecaster):
+    """Online Updating Ensemble of forecasters
+
+    Parameters
+    ----------
+    ensemble_algorithm : ensemble algorithm
+    forecasters : list of (str, estimator) tuples
+    n_jobs : int or None, optional (default=None)
+        The number of jobs to run in parallel for fit. None means 1 unless
+        in a joblib.parallel_backend context.
+        -1 means using all processors.
+    """
+
+    _required_parameters = ["forecasters"]
+
+    def __init__(self, forecasters,
+                 ensemble_algorithm=None,
```

Comment:
the default will throw an error? 
(in `fit`, attempted call of `_update` method of `None`)
Default should be sensible and not throw errors.
Suggestion: uniform, or make mandatory user input (no default).

Reply:
`fit` never makes a call to `_update` the default will be the uniform weights over all forecasters set in the `init` and works for `fit` and `predict`.

Reply:
There is a test in `sktime.forecasting.tests.test_online` that shows that this won't error out because the default behaviour is just to average the predictions. This also is why there is no average ensemble algorithm.